src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/CMakeLists.txt

```diff
@@ -17,6 +18,8 @@ spectre_target_headers(
   HEADERS
   GaussianPlusConstant.hpp
   DampingFunction.hpp
+  RegisterDerivedWithCharm.hpp
```

Comment:
Not related to the changes of this PR, but, since you are editing this file, it wouldn't hurt to fix the alphabetical order of these headers. Up to you.

Reply:
Thanks for catching this! I meant to fix this myself. Not sure how to do it, but would be great if someday this was caught automatically somehow.